README.md

```diff
@@ -3,47 +3,59 @@
 Helps settings prices and adds label to the issues accordingly.
 
 ## Usage
+
 Example of valid configuration:
+
 ```yml
- - plugin: https://ubiquibot-assistive-pricing.ubq.fi
-   type: github
-   with:
-     labels:
-       time:
-         - "Time: <1 Hour"
-         - "Time: <2 Hours"
-         - "Time: <4 Hours"
-         - "Time: <1 Day"
-         - "Time: <1 Week"
-       priority:
-         - "Priority: 1 (Normal)"
-         - "Priority: 2 (Medium)"
-         - "Priority: 3 (High)"
-         - "Priority: 4 (Urgent)"
-         - "Priority: 5 (Emergency)"
-     basePriceMultiplier: 1
-     publicAccessControl:
-       setLabel: true
-       fundExternalClosedIssue: false
+- plugin: https://ubiquibot-assistive-pricing.ubq.fi
+  type: github
+  with:
+    labels:
+      time:
+        - "Time: <1 Hour"
+        - "Time: <2 Hours"
+        - "Time: <4 Hours"
+        - "Time: <1 Day"
+        - "Time: <1 Week"
+      priority:
+        - "Priority: 1 (Normal)"
+        - "Priority: 2 (Medium)"
+        - "Priority: 3 (High)"
+        - "Priority: 4 (Urgent)"
+        - "Priority: 5 (Emergency)"
+    basePriceMultiplier: 1
+    publicAccessControl:
+      setLabel: true
+      fundExternalClosedIssue: false
+    globalConfigUpdate:
+      enabled: true
```

Comment:
Would be better to have it disable if not present in the config I think.

Reply:
sorry @gentlementlegen do you mean `disable: true` or `enabled: false`?

Reply:
just having `globalConfigUpdate` present would imply that it is enabled, no need to add `enabled: true`, and if it is not present in the config then it implies it is disabled.